### PR TITLE
Fix vde_cryptcab so it compiles under MacOS

### PR DESCRIPTION
**Pull request by jguillaumes**
I hope 3 will be my lucky number.
See commit messages for details.

### CODE REVIEW

**Review by danielinux — CHANGES_REQUESTED** [review]
Please check indentation

**Comment by danielinux** [comment]
One last thing, please check the indentation of your code (memset and od= lines)

File: src/vde_cryptcab/vde_cryptcab_client.c [hunk 1]
```diff
@@ -96,9 +98,12 @@ rcv_challenge(struct datagram *pkt, struct peer *p)
 	}
 
 	close (fd);
-
 	memset(keyname + strlen(keyname) - 10, 'X', 6);
-	od = mkostemps(keyname, 4, O_RDWR | O_CREAT | O_TRUNC);
+#ifdef VDE_DARWIN
+    od = mkostemps(keyname, 4, O_EXLOCK);
```
Comment (danielinux) [inline src/vde_cryptcab/vde_cryptcab_client.c:103]: Should be tab not spaces
```
	memset(keyname + strlen(keyname) - 10, 'X', 6);
#ifdef VDE_DARWIN
	od = mkostemps(keyname, 4, O_EXLOCK);
#else
	od = mkostemps(keyname, 4, O_RDWR | O_CREAT | O_TRUNC);
#endif
```

**Review by danielinux — COMMENTED** [review]
Sorry for being pedantic about indentation. vde-2 uses tabs not spaces please fix the spacing of the od = line as suggested

**Comment by jguillaumes** [comment]
Oh well... 
I'm in a conference now, will fix it next weekend

**Comment by danielinux** [comment]
Closing, replaced by #45

**Pull request closed without merging**